src/pages/docs/theme/theme-api.mdx

```diff
@@ -3,7 +3,7 @@ Below is the TypeScript API for the theme. You can
 use this to define your theme and pass it to the provider.
 
 ## Default theme
-Default theme for each component you could find [here](?path=/story/components-theme--components)
+Default theme for each component you could find [here](https://storybook.reablocks.dev/?path=/story/components-theme--components).
```

Comment:
This doens't seem right

Reply:
That the link to storybook blocks
We don't have storybook blocks on website